### PR TITLE
Increased the limit on the number of messages

### PR DESCRIPTION
**Pull request by zmq810150896**
## 拉取/合并请求描述：(PR description)
添加AF_UNIX协议的消息数量限制(类似于网络滑动窗口大小)

#### 你的解决方案是什么 (what is your solution)
添加一个计数器对消息读写计数

#### 请提供验证的bsp和config (provide the config and bsp) 

- BSP:

- .config:

- action:

]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 代码是高质量的 Code in this PR is of high quality
- [x] 已经使用[formatting](https://github.com/mysterywolf/formatting) 等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

File: components/net/sal/include/sal_msg.h [hunk 1]
```diff
@@ -58,6 +58,9 @@ struct unix_conn
     int type;
     int proto;
 
+#ifdef SAL_USING_AF_UNIX
+    rt_atomic_t msg_count;
```
Comment (mysterywolf) [inline components/net/sal/include/sal_msg.h:62]: 在哪里用到了呢？

**Review by mysterywolf — CHANGES_REQUESTED** [review]

File: components/net/sal/include/sal_msg.h (hunk 1 above)
Comment (zmq810150896) [reply components/net/sal/include/sal_msg.h:62]: socket中的AF_UNIX协议中

**Review by geniusgogo — APPROVED** [review]

File: components/net/sal/include/sal_msg.h (hunk 1 above)
Comment (mysterywolf) [reply components/net/sal/include/sal_msg.h:62]: 使用源码的链接贴一下？我搜了半天没搜到

Comment (zmq810150896) [reply components/net/sal/include/sal_msg.h:62]: 这是ukernel里面的，还需要粘贴吗

Comment (mysterywolf) [reply components/net/sal/include/sal_msg.h:62]: 是的贴出来。链接发出来一下。

**Review by BernardXiong — APPROVED** [review]

**Review by mysterywolf — APPROVED** [review]

File: components/net/sal/include/sal_msg.h (hunk 1 above)
Comment (zmq810150896) [reply components/net/sal/include/sal_msg.h:62]: [mr链接](https://git.rt-thread.com/packages/ukernel/-/merge_requests/95/diffs)

**Pull request merged**